.gitattributes

```diff
@@ -1,3 +1,4 @@
-# Mark json files inside test/testdata as binary,
+# Mark everything inside test/testdata as binary,
 # so git won't replace \n with \r\n on windows.
-test/testdata/*.json binary
+test/testdata/* binary
+test/testdata/*.go text diff
```

Comment:
This means if we ever embed `build-fixtures.go` in a test fixture itself, it'll look different on windows and linux :thinking: 

Maybe we should use `binary diff` here, to have git keep the file as-is, but still use `diff` to show differences?

Reply:
Sure thing.